src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/rollup/model/dimension/Dimension.kt

```diff
@@ -40,6 +40,8 @@ abstract class Dimension(
 
     abstract fun toSourceBuilder(): CompositeValuesSourceBuilder<*>
 
+    abstract fun canBeRealizedInMappings(mappings: Map<String, Any>): Boolean
```

Comment:
Can you add a kdoc explaining what this means?

Reply:
will add